### DISCUSSION TITLE
RNA-Seq mission code

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#230] [Q&A]
when this section is followed from this page: https://www.biostarhandbook.com/books/rnaseq/rnaseq-mission-impossible.html

"Obtain the recipes
For each project you should download a separate copy of all the scripts. Edit and customize the scripts for your project.

# Obtain the biostar handbook rnaseq scripts.
curl -O http://data.biostarhandbook.com/books/rnaseq/code.tar.gz

# Unpack the code.
tar -xzvf code.tar.gz
A new directory called code will be created that contains the scripts.

Command line usage
The Makefile is located in code/mission-impossible.mk"

there is no makefile inside the code folder.  am I missing something?  should I be following something from the workflows chapter/book instead?  

thanks

**Comment by ialbert** [comment]
Ah, apologies, sometimes the wrong archive would get built that was missing the makefile.

It should be fixed and working now.

The RNA-Seq book is being rewritten with a newer approach and a more RStudio centric R code.
